docs/guides/server/update-compatibility.adoc

```diff
@@ -86,6 +86,8 @@ If you are upgrading to a new {project_name} version, this command must be execu
 Failure to meet these requirements results in an incorrect outcome.
 ====
 
+NOTE: If you do not use `--optimized` keep in mind that an `update` command may implicitly create or update an optimized build for you - if you are running the command from the same machine as a server instance, this may impact the next start of your server.
```

Comment:
Isn't this note also relevant to the `metadata` command?

Reply:
It's applicable to both update metadata and update check.

Reply:
Then I'd move this note a bit higher, below the first warning box in `Determining the update strategy for an updated configuration` chapter. The way it currently is it implies the note is relevant only to the `check` command.

Reply:
@vmuzikar just to double check, you are in favor of proceeding with these warnings instead of changing the --optimized logic to not use auto builds correct?

Reply:
Yes, having it just as a docs change is fine from my perspective. My suggestion in this thread is just moving the note/warning to a different place in the guide so it's clear it applies both to `metadata` and `check` commands.

Reply:
Ok, moved where the warning is at.